scripts/graphdocs.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
```

Comment:
In "support channel policies in LN edges":

Now that we have two script to generate these docs, might be worth adding a `just update-docs` helper script (in a followup or w/e)

Reply:
Considered this but seems unlikely that both docs would need to be updated in the same commit?